apis/apiextensions/v1/xrd_webhooks_test.go

```diff
@@ -17,156 +17,7 @@ limitations under the License.
 package v1
 
 import (
-	"testing"
-
-	"github.com/google/go-cmp/cmp"
-	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
-
-	"github.com/crossplane/crossplane-runtime/pkg/errors"
-	"github.com/crossplane/crossplane-runtime/pkg/test"
 )
 
 var _ admission.Validator = &CompositeResourceDefinition{}
-
-func TestValidateUpdate(t *testing.T) {
```

Comment:
we are removing unit tests for these validation tests, but not introducing e2e tests for the same.